documentation/release_notes.rst

```diff
@@ -11,14 +11,71 @@ creating efficient heterogeneous applications.
 New in 2022.10.0
 ================
 
+Deprecation Notices
+-------------------
+The ``ONEDPL_USE_AOT_COMPILATION`` and ``ONEDPL_AOT_ARCH`` CMake options are deprecated and will be removed in a future
+release. Please use the relevant compiler flags to enable this feature.
+
+New Features
+------------
+- Added parallel range algorithms in ``namespace oneapi::dpl::ranges``: ``set_intersection``, ``set_union``,
+  ``set_difference``, ``set_symmetric_difference``, ``includes``, ``unique``, ``unique_copy``, ``destroy``,
+  ``uninitialized_fill``, ``uninitialized_move``, ``uninitialized_copy``, ``uninitialized_value_construct``,
+  ``uninitialized_default_construct``, ``reverse``, ``reverse_copy``, ``swap_ranges``. These algorithms operate with
+  C++20 random access ranges.
+- Improved performance of ``oneapi::dpl::experimental::kt::gpu::inclusive_scan`` and added support for binary operator
+  and type combinations which do not have a SYCL known identity.
+- Improved performance of ``inclusive_scan_by_segment`` and ``exclusive_scan_by_segment`` with
+  device execution policies on GPUs.
+- Improved performance of set operation algorithms when using device policies: ``set_union``, ``set_difference``,
+  ``set_intersection``, ``set_symmetric_difference``.
+- Improved performance of parallel_find_or–based algorithms on Intel® Arc™ B-series GPU devices.
+- Removed requirement of GPU double support to use ``set_union``, ``set_difference``, ``set_intersection``, and
+  ``set_symmetric_difference`` on Windows operating systems.
+- Removed copy constructible requirements from value type for reduce and transform_reduce algorithms when used with a
+  host policy, and default constructible requirements from value type for reduce and transform_reduce when used with an execution policy.
```

Comment:
@danhoeflinger at the end of the line here is "an execution policy" a host or device policy?  Is it any policy?

Reply:
After offline discussion fixing this to indicate all policies are impacted.